client/src/App.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react';
+import React, { Component, useState } from 'react';
```

Comment:
all looks great. Is the `useState` hook being used?

Reply:
From docs: 
> Hooks don’t work inside classes. But you can use them instead of writing classes.

https://reactjs.org/docs/hooks-state.html

I'll update this in my repo to create a functional component instead of a class to leverage the `useState` hook